compiler/rustc_ast/src/util/comments.rs

```diff
@@ -24,6 +24,14 @@ pub struct Comment {
     pub pos: BytePos,
 }
 
+/// A fast conservative estimate on whether the string can contain documentation links.
+/// A pair of square brackets `[]` must exist in the string, but we only search for the
+/// opening bracket because brackets always go in pairs in practice.
+#[inline]
+pub fn may_have_doc_links(s: &str) -> bool {
+    s.contains('[')
```

Comment:
Originally you checked for both `[` and `]`. Is there a reason why it's not the case anymore? (this is pure curiosity)

Reply:
I left a comment about this above.
I collected some statistics from the standard library and the `[]` pair appeared exactly the same number of times as a single opening bracket `[` (about 4000 times, IIRC).
So if we are doing a conservative estimate we don't even need to check for the closing bracket, it's pretty much guaranteed to be there.

Reply:
Sorry, I didn't see the change in the doc comments. Thanks for your answer. :)